source/network/ServerSideNetworkHandler.cpp

```diff
@@ -479,44 +479,44 @@ void ServerSideNetworkHandler::commandTP(OnlinePlayer* player, const std::vector
 {
 	if (!m_pLevel)
 		return;
-
+    
```

Comment:
lmao, add spaces instead of removing

Reply:
Xcode loves using spaces instead of tabs. I hit enter, it did that.